lib/Frontend/Frontend.cpp

```diff
@@ -942,11 +954,8 @@ bool CompilerInstance::loadPartialModulesAndImplicitImports(
   assert(DefaultSerializedLoader && "Expected module loader in Compiler Instance");
   FrontendStatsTracer tracer(getStatsReporter(),
                              "load-partial-modules-and-implicit-imports");
-  // Force loading implicit imports. This is currently needed to allow
-  // deserialization to resolve cross references into bridging headers.
-  // FIXME: Once deserialization loads all the modules it needs for cross
-  // references, this can be removed.
-  (void)mod->getImplicitImports();
+
+  swift::forceImplicitImportLoading(mod);
```

Comment:
IIRC this hack was only required to load in the bridging header before deserialization, as all the other imports will be loaded when evaluating the module file's dependencies. Is `(void)mod->getImplicitImports();` no longer sufficient for this hack?

Reply:
It's possible that it's still sufficient—I'll check.

Reply:
It looks like it does still suffice; I'll go back to that, since it lets me get rid of some code.